<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar</class>
 <widget class="QDialog" name="agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Productos</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>volumen</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="sp_volumen"/>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QRadioButton" name="rd_educativo">
     <property name="text">
      <string>educativos</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>precio</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="sp_precio"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>peso</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="sp_peso"/>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="rd_alcohol">
     <property name="text">
      <string>alcoholicos</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>tasa</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="sp_tasa"/>
   </item>
   <item row="4" column="2">
    <widget class="QRadioButton" name="rd_lujo">
     <property name="text">
      <string>de lujo</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>cliente</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="cb_cliente"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Agregar Producto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
